app/api/files/jsRoutes.js

```diff
@@ -76,6 +86,11 @@ export default app => {
           socket(req).emit('documentProcessed', newEntity.sharedId);
         });
       }
+
+      if (req.body.email) {
```

Comment:
This 'if' suggests the possibility of the `email` not being present on the request, yet every test scenario injects the body.email key.  Please, extend the test cases to a scenario where that is not the case.

Reply:
Done